agent/config/config.go

```diff
@@ -570,6 +570,7 @@ func environmentConfig() (Config, error) {
 		SpotInstanceDrainingEnabled:         parseBooleanDefaultFalseConfig("ECS_ENABLE_SPOT_INSTANCE_DRAINING"),
 		GMSACapable:                         parseGMSACapability(),
 		VolumePluginCapabilities:            parseVolumePluginCapabilities(),
+		ExecAgentLogDir:                     os.Getenv("ECS_EXEC_AGENT_LOG_DIR"),
```

Comment:
can you add a TODO for later to add this env var to README?

Reply:
Added